networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/ActiveEth2Network.java

```diff
@@ -197,20 +249,64 @@ private synchronized void startup() {
 
     processedAttestationSubscriptionProvider.subscribe(attestationGossipManager::onNewAttestation);
     processedAttestationSubscriptionProvider.subscribe(aggregateGossipManager::onNewAggregate);
+
+    setTopicScoringParams();
+  }
+
+  private void setTopicScoringParams() {
+    final GossipTopicsScoringConfig.Builder builder = GossipTopicsScoringConfig.builder();
+    gossipConfigurator.configureAllTopics(builder, getEth2Context());
+    discoveryNetwork.updateGossipTopicScoring(builder.build());
```

Comment:
Is there a reason `gossipConfigurator.configureAllTopics` doesn't just create the `GossipTopicsScoringConfig` itself rather than needing a builder passed in?

Reply:
It's more generic to operate on the builder and we call to this method with an existing builder within `GossipTopicsScoringConfig`.  Added some default interface methods that construct the builder and pass it in.